commands/cli.go

```diff
@@ -135,3 +142,107 @@ func awsNuke(c *cli.Context) error {
 
 	return nil
 }
+
+func awsDefaults(c *cli.Context) error {
+	logging.Logger.Infoln("Identifying enabled regions")
+	regions, err := aws.GetEnabledRegions()
+	if err != nil {
+		return errors.WithStackTrace(err)
+	}
+	for _, region := range regions {
+		logging.Logger.Infof("Found enabled region %s", region)
+	}
+
+	err = defaultVpcs(c, regions)
```

Comment:
NIT: the function name should probably indicate that it will nuke the default VPCs. At a glance, it is not exactly clear what this is expected to do.